### ISSUE TITLE
No cursor in Wayland applications

### ISSUE THREAD

**Issue by Bu156** [maltejur/discord-screenaudio#103]
When sharing desktop or Wayland windows, the cursor is not shown on the stream, but for XWayland apps, the cursor does show. This is different from the behaviour with regular Chromium, which shows the cursor correctly in all situations.

**Comment by Bu156** [comment]
Of note is that this happens on both my desktop (Fedora 37 GNOME Wayland) and my friend's (Ubuntu 22.10 GNOME Wayland)

**Comment by maltejur** [comment]
Hm, after a bit of testing this seems to be a general qtwebengine issue. I don't know how much I can do there

**Comment by Bu156** [comment]
Yeah, I've searched a bit more and found out that it's an issue with WebRTC, so as it stands it currently affects Firefox as well for me.

**Comment by Bu156** [comment]
This does not happen for me on wlroots based compositors (tested on Sway and Hyrpland) so it seems to be a GNOME+WebRTC issue
